### PR TITLE
Header v3 expanded dropdown styling

### PR DESCRIPTION
**Pull request by zeftilldeath**
I've been working on improving the styling of the dropdowns in the nav.

# They now look like this:
screen shot 2017-10-20 at 15 49 49

interaction

**Some notes on what I've done**
1. Dividing lines with top aligned text 
2. An active state for your chosen edition (Thank you @NataliaLKB) 
3. Slight hover on dropdown
4. Lots of spacing adjustments
5. Removed drop shadow and added opacity 1px border 

Currently, the selected item will always be at the top, I'm not sure if that's the best option?

### CODE REVIEW

File: static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss [hunk 1]
```diff
@@ -8,52 +8,64 @@
 .dropdown-menu {
     display: none;
     position: absolute;
-    top: 29px; //visually this sits right under the text
-    left: -($gs-gutter / 2);
-    width: $gs-column-width * 3;
+    top: $gs-baseline * 2 + $gs-baseline / 2;
+    right: -8px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss]: comment?

File: static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss [hunk 2]
```diff
@@ -8,52 +8,64 @@
...
+        border-bottom: $gs-baseline / 2 + $gs-baseline / 4 - 1 solid #ffffff;
+        border-left: $gs-baseline / 2 + $gs-baseline / 4 - 1 solid transparent;
+        border-right: $gs-baseline / 2 + $gs-baseline / 4 - 1 solid transparent;
     }
 }
 
 .dropdown-menu--open {
     display: block;
+
+    & ~ .top-bar__item--dropdown {
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss]: Should this happen for the fallback as well?

File: static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss [hunk 3]
```diff
@@ -8,52 +8,64 @@
...
     cursor: pointer;
     display: block;
+    letter-spacing: .01em;
     outline: none;
-    padding: $gs-baseline + 1 $gs-gutter $gs-baseline;
+    padding: ($gs-baseline / 2 + 1) $gs-gutter ($gs-baseline + $gs-baseline / 4) ($gs-baseline * 2 + $gs-baseline / 2);
     position: relative;
     box-sizing: border-box;
+    // Hides border on hover
     margin-top: -1px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss]: Would `backface-visibility: hidden;` work here?

File: static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss (hunk 2 above)
Comment (zeftilldeath) [reply static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss]: yes! Good spot. Checking it non-JS helped me spot a font weight issue too.

File: static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss (hunk 1 above)
Comment (zeftilldeath) [reply static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss]: I can do better : )

File: common/app/views/fragments/nav/editionPickerDropdown.scala.html [hunk 4]
```diff
@@ -14,32 +26,16 @@
...
-                        class="dropdown-menu__title">
+        @editionListItem(Edition(request), true)
 
-                        <span class="u-h">switch to the </span>
-                        @edition.displayName
-                    </a>
-                </li>
-            }
-        }
+        @Edition.others(request).map(e => editionListItem(e))
```
Comment (NataliaLKB) [inline common/app/views/fragments/nav/editionPickerDropdown.scala.html]: Can you see if this works:

```
@Edition.others(request).map(_ => editionListItem(_))
```

if so, lets do that instead.

File: common/app/views/fragments/nav/editionPickerDropdown.scala.html [hunk 5]
```diff
@@ -14,32 +26,16 @@
...
     </ul>
 
     <label for="edition-picker-toggle"
             class="top-bar__item top-bar__item--dropdown js-edition-picker-trigger"
             data-link-name="nav2 : topnav : edition-picker: toggle">
 
-        <span class="u-h">Current Edition: </span>
```
Comment (NataliaLKB) [inline common/app/views/fragments/nav/editionPickerDropdown.scala.html]: Can we keep this span? Its for screen readers.

File: static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss [hunk 6]
```diff
@@ -3,57 +3,77 @@
...
+        }
+    }
 }
 
 .dropdown-menu__title {
     @include fs-textSans(2);
     color: $neutral-1;
     cursor: pointer;
     display: block;
+    letter-spacing: .01em;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss]: `em` or `rem` ?

File: common/app/views/fragments/nav/editionPickerDropdown.scala.html (hunk 5 above)
Comment (zeftilldeath) [reply common/app/views/fragments/nav/editionPickerDropdown.scala.html]: Good spot. I didn't mean to remove that.

File: common/app/views/fragments/nav/editionPickerDropdown.scala.html (hunk 4 above)
Comment (zeftilldeath) [reply common/app/views/fragments/nav/editionPickerDropdown.scala.html]: Don't think it's working.

**Review by NataliaLKB — APPROVED** [review]
🎉

**Pull request merged**